scripts/template_matcher.py

```diff
@@ -8,6 +9,17 @@
 class TemplateMatcher(object):
 
     def __init__ (self, images, min_match_count=10, good_thresh=0.7):
+        """ Store template images, create the sift and matching objects
+            which will be used for processing images
+
+            images: dictionary of "template name":"template filename" pairs,
+                which is used to find and store the actual images
+            min_match_count: Designed to be the minimum number of keypoint
+                matches for a template to be considered a possible
+                match
+            good_thresh: Used for homography transformation - determines whether
```

Comment:
Yay function documentation.  For future reference, I like how Paul named his `good_thresh` variable as `ratio_threshold`, since it is the special parameter talked about in Lowe's Ratio Test.  Let me know if you don't know what I'm talking about by Lowe's Ratio Test, cuz I think it's pretty fascinating to read about in Lowe's SIFT paper